### PR TITLE
Update the issue links in the 2201.3.1 release note

### PR DESCRIPTION
**Pull request by praneesha**
## Purpose
Update the issue links in the 2201.3.1 release note.
> Fixes #

## Checklist

- [ ] **Page addition**
  - [ ] Add `permalink` to pages.

- [ ] **Page removal**
  - [ ] Remove entry from corresponding left nav YAML file.
  - [ ] Add `redirect_from` on the alternative page.
  - [ ] If no alternative page, add redirection on the `redirections.js ` file.

- [ ] **Page rename**
  - [ ] Add front-matter `redirect_from`.
  - [ ] Add front-matter `redirect_to:` (if applicable).

- [ ] **Page restrcuture**
  - [ ] Add `permalink` to pages.
  - [ ] Add front-matter `redirect_from`.
  - [ ] Add front-matter `redirect_to:` (if applicable).

### CODE REVIEW

File: downloads/swan-lake-release-notes/swan-lake-2201.3.1/RELEASE_NOTE.md [hunk 1]
```diff
@@ -30,37 +30,31 @@ However, if you are using a version below 2201.0.0 (Swan Lake) and if you alread
...
 
-If you have not installed Ballerina, then download the [installers](/downloads/#swanlake) to install.
+If you have not installed Ballerina, then, download the [installers](/downloads/#swanlake) to install.
 
 ## Language updates
 
 ### Bug fixes
 
-To view bug fixes, see the [GitHub milestone for 2201.3.1 (Swan Lake)](https://github.com/ballerina-platform/ballerina-lang/issues?q=is%3Aissue+milestone%3A2201.3.1+label%3AType%2FBug+is%3Aclosed).
+- [https://github.com/ballerina-platform/ballerina-lang/issues/38583](https://github.com/ballerina-platform/ballerina-lang/issues/38583)
```
Comment (anupama-pathirage) [inline downloads/swan-lake-release-notes/swan-lake-2201.3.1/RELEASE_NOTE.md:39]: Add the issue titles instead of links for the text. Check for all places.

**Review by sm1990 — APPROVED** [review]

File: downloads/swan-lake-release-notes/swan-lake-2201.3.1/RELEASE_NOTE.md [hunk 2]
```diff
@@ -30,37 +30,31 @@ However, if you are using a version below 2201.0.0 (Swan Lake) and if you alread
...
+- [https://github.com/ballerina-platform/ballerina-lang/issues/38518](https://github.com/ballerina-platform/ballerina-lang/issues/38518)
+- [https://github.com/ballerina-platform/ballerina-lang/issues/36270](https://github.com/ballerina-platform/ballerina-lang/issues/36270)
 
 ## Standard library updates
 
-### Improvements
-
 ### Bug fixes
 
 To view bug fixes, see the [GitHub milestone for 2201.3.1 (Swan Lake)](https://github.com/ballerina-platform/ballerina-standard-library/issues?q=is%3Aissue+milestone%3A2201.3.1+label%3AType%2FBug+is%3Aclosed).
```
Comment (anupama-pathirage) [inline downloads/swan-lake-release-notes/swan-lake-2201.3.1/RELEASE_NOTE.md:47]: To make it consistent , shall we list down issues separately for stdlib issues as well?

Comment (praneesha) [reply downloads/swan-lake-release-notes/swan-lake-2201.3.1/RELEASE_NOTE.md]: Done.

File: downloads/swan-lake-release-notes/swan-lake-2201.3.1/RELEASE_NOTE.md (hunk 1 above)
Comment (praneesha) [reply downloads/swan-lake-release-notes/swan-lake-2201.3.1/RELEASE_NOTE.md:39]: Fixed via https://github.com/ballerina-platform/ballerina-dev-website/commit/eeadba544e44a9ab4a4dd8421656f2efc50d1b57.

**Pull request merged**